SparkPost/support-docs
# Updated analytics report md file and gifs

### What Changed

- Updated analytics report md file and gifs

### How To Test or Verify

- Check if docs are matching the doc described in the task 
- Check if gifs are right 

### PR Checklist

Below are some checklists to follow for the correct procedure in different circumstance. The first list ("All PRs Checklist") should be followed for ALL PRs. The next 2 are additive to this list depending on what type of PR you are using.

For example: If you are submitting a content change to one of the support documents, your checklist would include the:

- "All PRs Checklist"
- AND the "Content Changes Checklist

If you are submitting a feature addition, enhancement, or bug fix, your checklist would include the:

- "All PRs Checklist"
- AND the "Development Changes Checklist"

#### All PRs Checklist


#### Content Changes Checklist


#### Development Changes Checklist (some checks are automatic github actions and will not be listed here. ie. "all tests pass")
